File: iloveruby/08-interesante/17-java/Customer.java
// Ejemplo de https://medium.com/@Nicholson85/method-overloading-in-ruby-22a169e704c7
public class Customer {
	public static void DisplayInfo(String firstName){
		Console.WriteLine("Their name is " + firstName);
	}

	public static void DisplayInfo(String firstName, String lastName){
		Console.WriteLine("Their name is " + firstName + " " + lastName);
	}

	public static void DisplayInfo(String firstName, String lastName, String email){
		Console.WriteLine("Their name is " + firstName + " " + lastName + " and their email address is " + email);
	}
}

File: iloveruby/09-otros-lenguajes/java/customer.rb
# Ejemplo de dvarrui
class Customer
	def display_info(input = {})
		output = ''
		output << "Their name is #{input[:firstname]}" if input[:firstname]
		output << " #{input[:lastname]}" if input[:lastname]
		output << " and their email address is #{input[:email]}" if input[:email]
		puts output
	end
end
